autoload/test/python.vim

```diff
@@ -1,4 +1,4 @@
 let test#python#patterns = {
   \ 'test':      ['\v^\s*%(async )?def (test_\w+)'],
-  \ 'namespace': ['\v^\s*class (Test\w+)|(\w+)\(unittest\.TestCase\)'],
+  \ 'namespace': ['\v^\s*class (Test\w+|\w+)'],
```

Comment:
Note that this regex is equivalent to `'\v^\s*class (\w+)'`.

Reply:
Given the issues, do you think we should just simplify the regex down to that and treat classes defined instead a test block as more an edge case?